### DISCUSSION TITLE
Failed to Assign Domain:The chosen alias "**.vercel.app" is already in use

### DISCUSSION THREAD

**Discussion by ghost** [vercel/vercel#5023] [Help] [answered]
every time my deploy runs i am getting this error. is it because I removed the vercel.app domain after adding a custom domain? i thought it will affect SEO if I have two domains. should i add it back? will it affect SEO? 

or is there another reason should i follow?

**Comment by styfle** [comment, marked as answer]
Hi @alhaqi 

The `vercel.app` suffix is reserved for project names as well as user names that exist on the Vercel platform.

This error is expected when the name is already in use. Try selecting a different domain name instead.

If you think this is incorrect, submit a [support request](https://vercel.com/support/request) and we'll see why its reserved.
